### ISSUE TITLE
No

### ISSUE THREAD

**Issue by se7enthsinvii** [SubzeroGPT/CODM-007#1]

**Comment by SubzeroGPT** [comment]
Yeah because I haven't been working on it under this repo I have been testing it on a private repo..

It's a AI Operating System that can create, design, write, build anything and it's called Kai OS..

**Comment by SubzeroGPT** [comment]
I been working on other projects I have ADHD so hard to focus working on using electromagnets and electricity to power a new machine that can warp small areas of space and time..
